src/Common/Elf.cpp

```diff
@@ -119,13 +119,32 @@ std::optional<Elf::Section> Elf::findSectionByName(const char * name) const
 
 String Elf::getBuildID() const
 {
+    /// Section headers are the first choice for a not loaded file
```

Comment:
What does it mean "not loaded file"?

Reply:
I tried to say "for a file that is not loaded in memory", while it is not perfect.
Do you mind changing to less ambitious "Section headers are the first choice for a debuginfo file"?
Actually it is the only Elf::getBuildID() use case.